File: app/src/main/java/com/acterics/healthmonitor/ui/drawerfragments/SettingsFragment.java
package com.acterics.healthmonitor.ui.drawerfragments;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.acterics.healthmonitor.R;
import com.acterics.healthmonitor.base.BaseFragment;

import butterknife.ButterKnife;

/**
 * Created by oleg on 12.05.17.
 */

public class SettingsFragment extends BaseFragment {

    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        View view = inflater.inflate(R.layout.fragment_settings, container, false);
        ButterKnife.bind(this, view);

        return view;
    }

    @Override
    public boolean onBackPressed() {
        return false;
    }
}

File: app/src/main/java/com/acterics/healthmonitor/ui/drawerfragments/SettingsFragment.kt
package com.acterics.healthmonitor.ui.drawerfragments

import android.os.Bundle
import android.support.v4.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import butterknife.ButterKnife
import com.acterics.healthmonitor.R

/**
 * Created by root on 25.05.17.
 */

class SettingsFragment : Fragment() {



    override fun onCreateView(inflater: LayoutInflater?, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        val view = inflater?.inflate(R.layout.fragment_settings, container, false)



        return view
    }

}